pyproject.toml

```diff
@@ -1,8 +1,8 @@
 # This file is protected via CODEOWNERS
 
 [build-system]
-requires = ["flit_core >=3.2,<4"]
-build-backend = "flit_core.buildapi"
+requires = ["hatchling>=1.6.0"]
```

Comment:
I'm not sure how hatchling does it's versioning, but are you saying with this that there won't be any breaking changes into the future?

Reply:
With the minimal set of features below, yes definitely never! I can cap if you want though.